ui/.nvmrc

```diff
@@ -1 +1 @@
-v21.6.1
+21.7.3
```

Comment:
@ggjulio looks like Renovate wants to stay within the same minor version series. Should we just manually bump this up?

Reply:
@crenshaw-dev yes, I forgot to bump it in the PR